packages/edit-post/src/components/layout/index.js

```diff
@@ -84,27 +84,17 @@ const DESIGN_POST_TYPES = [
 	'wp_navigation',
 ];
 
-function useEditorStyles() {
-	const {
-		hasThemeStyleSupport,
-		editorSettings,
-		isZoomedOutView,
-		renderingMode,
-		postType,
-	} = useSelect( ( select ) => {
-		const { __unstableGetEditorMode } = select( blockEditorStore );
-		const { getCurrentPostType, getRenderingMode } = select( editorStore );
-		const _postType = getCurrentPostType();
+function useEditorStyles( ...additionalStyles ) {
+	const { hasThemeStyleSupport, editorSettings } = useSelect( ( select ) => {
 		return {
 			hasThemeStyleSupport:
 				select( editPostStore ).isFeatureActive( 'themeStyles' ),
 			editorSettings: select( editorStore ).getEditorSettings(),
-			isZoomedOutView: __unstableGetEditorMode() === 'zoom-out',
-			renderingMode: getRenderingMode(),
-			postType: _postType,
 		};
 	}, [] );
 
+	const addedStyles = additionalStyles.join( '\n' );
```

Comment:
Nit: Currently there is only one style for the padding appender, so no line break will occur, but is the line break itself even necessary?

Reply:
No, it’s not necessary. Passing something as a separator is required to avoid the default comma. I suppose I opted for the line break as a formatting nicety despite the fact that it’d be a rare case that anyone is actually looking at the concatenated styles (and that as of yet there’s not more than one style so it’s moot for now). Here’s an example of what it looks like inspecting the element if there were more than one style.

empty string separator:
<img width="795" alt="image" src="https://github.com/user-attachments/assets/af4bda3d-629c-4dbb-b589-5b2b4554feaa">

`\n` seperator:
<img width="795" alt="image" src="https://github.com/user-attachments/assets/10156294-7a94-424a-b75e-270a30ba7f3f">

I don’t mind making it an empty string or space if you’d opine it should be.

Reply:
I don't have any strong opinions about it, so I think it's fine to merge it as is 👍